src/main/java/frc/robot/subsystems/shooter/ShooterIO.java

```diff
@@ -0,0 +1,41 @@
+package frc.robot.subsystems.shooter;
+
+import org.littletonrobotics.junction.AutoLog;
+
+public interface ShooterIO {
+
+  @AutoLog
+  public static class ShooterIOInputs {
```

Comment:
add motor voltage for each motor as this is helpful when debugging; also add closed loop reference slope for the angle motor